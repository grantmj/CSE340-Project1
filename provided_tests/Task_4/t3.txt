TASKS 
    1 4

POLY 
    F(z,y,x) = x y y  + z^2 x +  x y^2  + x z^2;

EXECUTE 
    INPUT x;

INPUTS 
    1