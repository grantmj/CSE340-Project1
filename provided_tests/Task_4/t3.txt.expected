POLY - COMBINED MONOMIAL LISTS
	F(z,y,x) = 2 y^2 x + 2 z^2 x;
